features/netsocket/cellular/cellular_driver_l3ip.cpp

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Update copyright to standard mbed-os (not freescale)

Reply:
done